Update list of operation result codes to conform to CAP-38 (#603)
stellar/new-docs: content/docs/start/list-of-operations.mdx

| CHANGE_TRUST_INVALID_LIMIT | -3 | The `limit` is not sufficient to hold the current balance of the trustline and still satisfy its buying liabilities. This error occurs when attempting to remove a trustline with a non-zero asset balance. |
| CHANGE_TRUST_LOW_RESERVE | -4 | This account does not have enough XLM to satisfy the minimum XLM reserve increase caused by adding a subentry and still satisfy its XLM selling liabilities. For every new trustline added to an account, the minimum reserve of XLM that account must hold increases. |
| CHANGE_TRUST_SELF_NOT_ALLOWED | -5 | The source account attempted to create a trustline for itself, which is not allowed. |
| CHANGE_TRUST_TRUST_LINE_MISSING | -6 | The asset trustline is missing for the liquidity pool |
| CHANGE_TRUST_CANNOT_DELETE | -7 | The asset trustline is still referenced by a liquidity pool |

> | CHANGE_TRUST_NOT_AUTH_MAINTAIN_LIABILITIES | -8 | The asset trustline is deauthorized |

Reviewer: Missing period at end of sentences (x3)

Author: Nice catch, fixed

| REVOKE_SPONSORSHIP_NOT_SPONSOR | -2 | If the ledgerEntry/signer is sponsored, then the source account must be the sponsor. If the ledgerEntry/signer is not sponsored, the source account must be the owner. This error will be thrown otherwise. |
| REVOKE_SPONSORSHIP_LOW_RESERVE | -3 | The sponsored account does not have enough XLM to satisfy the minimum balance increase caused by revoking sponsorship on a ledgerEntry/signer it owns, or the sponsor of the source account doesn't have enough XLM to satisfy the minimum balance increase caused by sponsoring a transferred ledgerEntry/signer. |
| REVOKE_SPONSORSHIP_ONLY_TRANSFERABLE | -4 | Sponsorship cannot be removed from this ledgerEntry. This error will happen if the user tries to remove the sponsorship from a ClaimableBalanceEntry. |

> | REVOKE_SPONSORSHIP_MALFORMED | -5 | One or more of the parameters to the operation were invalid. |

Reviewer: ```suggestion
| REVOKE_SPONSORSHIP_MALFORMED | -5 | One or more of the inputs to the operation was malformed. |
```

| Error | Code | Description |
| --- | --- | --- |
| LIQUIDITY_POOL_DEPOSIT_MALFORMED | -1 | One or more of the inputs to the operation was malformed. |
| LIQUIDITY_POOL_DEPOSIT_NO_TRUST | -2 | No trustline exists for one of the assets being deposited. |
| LIQUIDITY_POOL_DEPOSIT_NOT_AUTHORIZED | -3 | The account does not have authorization for one of the assets. |

> | LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED | -4 | There's not enough balance of one of the assets to perform the deposit. |

Reviewer: ```suggestion
| LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED | -4 | There is not enough balance of one of the assets to perform the deposit. |
```

| Error | Code | Description |
| --- | --- | --- |
| LIQUIDITY_POOL_WITHDRAW_MALFORMED | -1 | One or more of the inputs to the operation was malformed. |

> | LIQUIDITY_POOL_WITHDRAW_NO_TRUST | -2 | There's no trustline for one of the assets. |

Reviewer: ```suggestion
| LIQUIDITY_POOL_WITHDRAW_NO_TRUST | -2 | There is no trustline for one of the assets. |
```
